packages/cactus-common/src/test/typescript/unit/strings/replace-all.ts

```diff
@@ -0,0 +1,14 @@
+import test, { Test } from "tape";
```

Comment:
@gabrielsoaresm94 Delete this in favor of `packages/cactus-common/src/test/typescript/unit/strings/replace-all.test.ts` ?

Reply:
@petermetz I did not merge automatically, because of that some files remained, but I finished correcting it.